src/core/server/interface/Interaction.ts

```diff
@@ -8,5 +8,6 @@ export interface Interaction {
     type?: string;
     dimension?: number;
     callback?: Function;
+    drawMarker?: boolean;
```

Comment:
I believe this should always be set to true when adding an interaction to not prevent current `default` behavior. This can be done inside of the `add` function.